filnet/bootstrap.go

```diff
@@ -94,28 +94,28 @@ func (b *Bootstrapper) Stop() {
 // bootstrap does the actual work. If the number of connected peers
 // has fallen below b.MinPeerThreshold it will attempt to connect to
 // a random subset of its bootstrap peers.
-func (b *Bootstrapper) bootstrap(ctx context.Context, currentPeers []peer.ID) {
+func (b *Bootstrapper) bootstrap(currentPeers []peer.ID) {
 	peersNeeded := b.MinPeerThreshold - len(currentPeers)
 	if peersNeeded < 1 {
 		return
 	}
 
-	ctx, cancel := context.WithTimeout(b.ctx, b.ConnectionTimeout)
 	var wg sync.WaitGroup
 	defer func() {
 		wg.Wait()
```

Comment:
we have to wait until we connect to the bootstrap peers before starting dht routing?